docs/standard/serialization/version-tolerant-serialization.md

```diff
@@ -15,36 +15,36 @@ helpviewer_keywords:
+
+The VTS features are enabled when using the <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter>. Additionally, all features except extraneous data tolerance are also enabled when using the <xref:System.Runtime.Serialization.Formatters.Soap.SoapFormatter>. For more information about using these classes for serialization, see [Binary Serialization](binary-serialization.md).
+
 [!INCLUDE [binary-serialization-warning](../../../includes/binary-serialization-warning.md)]
 
-## Feature list  
- The set of features includes the following:  
-  
-- Tolerance of extraneous or unexpected data. This enables newer versions of the type to send data to older versions.  
-  
-- Tolerance of missing optional data. This enables older versions to send data to newer versions.  
-  
-- Serialization callbacks. This enables intelligent default value setting in cases where data is missing.  
-  
- In addition, there is a feature for declaring when a new optional field has been added. This is the <xref:System.Runtime.Serialization.OptionalFieldAttribute.VersionAdded%2A> property of the <xref:System.Runtime.Serialization.OptionalFieldAttribute> attribute.  
-  
- These features are discussed in greater detail below.  
-  
-## Tolerance of extraneous or unexpected data  
- In the past, during deserialization, any extraneous or unexpected data caused exceptions to be thrown. With VTS, in the same situation, any extraneous or unexpected data is ignored instead of causing exceptions to be thrown. This enables applications that use newer versions of a type (that is, a version that includes more fields) to send information to applications that expect older versions of the same type.  
-  
- In the following example, the extra data contained in the `CountryField` of version 2.0 of the `Address` class is ignored when an older application deserializes the newer version.  
-  
+## Feature list
+
+The set of features includes the following:
+
+- Tolerance of extraneous or unexpected data. This enables newer versions of the type to send data to older versions.
+- Tolerance of missing optional data. This enables older versions to send data to newer versions.
+- Serialization callbacks. This enables intelligent default value setting in cases where data is missing.
+
+In addition, there is a feature for declaring when a new optional field has been added. This is the <xref:System.Runtime.Serialization.OptionalFieldAttribute.VersionAdded%2A> property of the <xref:System.Runtime.Serialization.OptionalFieldAttribute> attribute.
+
+These features are discussed in greater detail in the following sections.
+
+### Tolerance of extraneous or unexpected data
```

Comment:
Made this header and other headers one-level down as they're sub items from the `Feature list`  header.